main.go

```diff
@@ -19,12 +20,14 @@ var resourceManagerURL = flag.String("resource-manager-url", "http://localhost:8
 var historyServerURL = flag.String("history-server-url", "http://localhost:19888", "The HTTP URL to access the history server.")
 var proxyServerURL = flag.String("proxy-server-url", "", "The HTTP URL to access the proxy server, if separate from the resource manager.")
 var namenodeAddress = flag.String("namenode-address", "localhost:9000", "The host:port to access the Namenode metadata service.")
+var statsdAddress = flag.String("statsd-address", "localhost:8200", "Optional: the host:port to access the statsd metrics service.")
```

Comment:
this should maybe not have a default value

Reply:
what happens if the statsd isn't running? does it fail? or silently proceed?

Reply:
see PR message, it seems to be fine. 
but another concern might be connections to another process listening on the default port.